integrations/amazon_bedrock/src/haystack_integrations/components/generators/amazon_bedrock/generator.py

```diff
@@ -193,21 +222,37 @@ def invoke(self, *args, **kwargs):
 
         return responses
 
-    @component.output_types(replies=List[str], metadata=List[Dict[str, Any]])
+    @component.output_types(replies=List[str])
```

Comment:
@davidsbatista we missed this one during previous review, there is not metadata output